pyartcd/pyartcd/pipelines/build_microshift_bootc.py

```diff
@@ -81,6 +82,10 @@ async def run(self):
             data_path=self._doozer_env_vars["DOOZER_DATA_PATH"]
         )
         self.assembly_type = get_assembly_type(self.releases_config, self.assembly)
+
+        microshift_nvrs = await get_microshift_builds(self.group, self.assembly, env=self._elliott_env_vars)
```

Comment:
for a named assembly, isn't the microshift nvr already pined in the assembly definition? why need to find builds again

Reply:
I suggest we can directly load the nvr from assembly, the get_microshift_builds() is gong to invoke find-builds and the end result is the same as pined rpm.